build-support/bin/generate_travis_yml.py

```diff
@@ -660,9 +662,7 @@ def integration_tests_v2(python_version: PythonVersion) -> Dict:
     **CACHE_NATIVE_ENGINE,
     "stage": Stage.test.value,
     "before_script": ["ulimit -n 8192"],
-    # NB: We also build `fs_util` in this shard to leverage having had compiled the engine. This
```

Comment:
The irony of this was that `cargo test` doesn't build in release mode, so we probably weren't gaining much actual benefit with regards to compilation. Oops!

Reply:
That would, uh, explain why Rust tests started taking consistently 40 minutes...